### PR TITLE
[GEOS-8257] Correct Y order for geopackage WPS tiles output.

### PR DESCRIPTION
**Pull request by bradh**
GeoPackage first tile is top left (flipped from mbtiles). This is already handled for
Map output, just incorrect for WPS tiles.

### CODE REVIEW

File: src/community/geopkg/src/test/java/org/geoserver/geopkg/wps/gs/GeoPackageProcessTileTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+package org.geoserver.geopkg.wps.gs;
```
Comment (jodygarnett) [inline src/community/geopkg/src/test/java/org/geoserver/geopkg/wps/gs/GeoPackageProcessTileTest.java]: header needed

File: src/community/geopkg/src/main/java/org/geoserver/geopkg/wps/GeoPackageProcess.java [hunk 2]
```diff
@@ -314,6 +314,7 @@ else if (tiles.getSldBody() != null) {
                }
                request.setFormat("none");
                Map formatOptions = new HashMap();
+               formatOptions.put("flipy", "true");
```
Comment (jodygarnett) [inline src/community/geopkg/src/main/java/org/geoserver/geopkg/wps/GeoPackageProcess.java]: I am wondering if we are fixing this issue (https://osgeo-org.atlassian.net/browse/GEOT-5582) and then running into trouble downstream here in geoserver

Comment (bradh) [reply src/community/geopkg/src/main/java/org/geoserver/geopkg/wps/GeoPackageProcess.java]: I don't think so. It isn't CRS dependent, more about how tile indices (in a TMS z/x/y sense) are used. GeoPackage does it one way, mbtiles the other.

**Comment by bradh** [comment]
I redid this as #2570 after the usual joy of rebase and github.

**Comment by bencaradocdavies** [comment]
@bradh AFAIK pull requests work by source branch name, so it should be possible to keep the same PR and use a force push onto your source branch.

**Pull request closed without merging**